test/Chirp.Razor.Tests/ChirpTests.cs

```diff
@@ -0,0 +1,102 @@
+using Chirp.Core.Classes;
+using Chirp.Core.Interfaces;
+using Chirp.Razor.Services;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.VisualStudio.TestPlatform.TestHost;
+
+namespace Chirp.Razor.Tests;
+
+public class UnitTests
```

Comment:
Looks great! :-)

I am a bit unsure whether these are "unit" tests, as we are initializing an instance of CheepService where many things can go wrong. Shouldn't a unit test just test one thing?

Reply:
I don't really know what else to unit test at this point if this isn't a unit test. I guess i can see how this might be more of an integration test, but i am not really sure.

maybe [9abb21f](https://github.com/ITU-BDSA2024-GROUP9/Chirp/pull/22/commits/9abb21fbdaeb9b5ad63ec45837d96b26bcde979e) is more correct?